src/main/cpp/PlayerLine.cpp

```diff
@@ -7,41 +7,54 @@ PlayerLine::PlayerLine()
 	
 }
 
-int PlayerLine::getNumPlayers() const 
+// PlayerLine::~PlayerLine() {
+// 	for(int i = 0; i < getNumPlayers(); i++) {
+// 		delete frontLine();
+// 		players.pop();
+// 	}
+// }
+
+int 
+PlayerLine::getNumPlayers() const
 {
 	return players.size();
 }
 
-void PlayerLine::addPlayer(string s, int num, int loc)
+void 
+PlayerLine::addPlayer(string s, int num, int loc)
 {
-	Player *p;
-	p = new Player(s,num,loc);
+	shared_ptr<Player> p(new Player(s,num,loc));
 	players.push(p);
 }
 
-void PlayerLine::addPlayer(Player *p) {
+void 
+PlayerLine::addPlayer(shared_ptr<Player> p) {
 	players.push(p);
 }
 
-void PlayerLine::nextTurn()
+void 
+PlayerLine::nextTurn()
 {
-	Player *p = players.front();
+	shared_ptr<Player> p = players.front();
```

Comment:
Can we keep it in the same form as the other shared_ptr initializers?
i.e. "shared_ptr<Player> p(players.front());"

Reply:
Done